src/cmds/versions/create.js

```diff
@@ -37,25 +37,24 @@ module.exports = class CreateVersionCommand {
       },
       {
         name: 'main',
-        type: String,
+        type: Boolean,
         description: 'Should this version be the primary (default) version for your project?',
       },
       {
         name: 'beta',
-        type: String,
+        type: Boolean,
         description: 'Is this version in beta?',
       },
       {
-        name: 'isPublic',
-        type: String,
+        name: 'public',
```

Comment:
This is going to be a breaking change for sure, but one I've wanted to do for a while because the `isPublic` argument, it being a string aside, was always out of place with the rest of these.

Reply:
I think having these as booleans makes a lot more sense here, but what does the usage look like? Is it just something like this?
```sh
rdme versions:create 2.0 --key=KEY --public --beta
```

Reply:
We'll also want to make sure our usage docs are updated: https://github.com/readmeio/rdme#create-a-new-version-1

Reply:
Yeah the usage is that.